packages/entity-server/src/__tests__/__integration__/fieldsAndFilters.test.ts

```diff
@@ -161,16 +161,20 @@ describe('fieldsAndFilters', () => {
   describe('hierarchy dependant fields', () => {
     it('parent_code depends on hierarchy used', async () => {
       const { body: rbEntity } = await app.get('hierarchy/redblue/BLUE', {
-        query: { fields: 'code,parent_code' },
+        query: { fields: 'code,parent_code,grandparent_code' },
       });
 
-      expect(rbEntity).toEqual({ code: 'BLUE', parent_code: 'PALLET' });
+      expect(rbEntity).toEqual({ code: 'BLUE', parent_code: 'PALLET', grandparent_code: 'KANTO' });
```

Comment:
So instead of extending an existing test, it might be nice to create a new one here. ie.
`grand_parent_code depends on hierarchy used`

And we might want to use an entity that changes grandparents between heirarchies, perhaps `LAVENDER` :+1: